src/core/agent/services/signifyNotificationService.ts

```diff
@@ -417,7 +421,23 @@ class SignifyNotificationService extends AgentService {
               });
               break;
             }
-
+            case OperationPendingRecordType.Oobi: {
+              if (!pendingOperation.metadata?.connectionId) {
```

Comment:
We don't need this `metadata` field in the pending operation record because in this function, we have the `operation` from Signify. So the connection ID is `operation.response.i` (like before)